OracleWebLogic/samples/wls-k8s-domain/README.md

```diff
@@ -0,0 +1,137 @@
+WebLogic Sample on Kubernetes with Shared Domain Home
+=========================================
+This sample extends the Oracle WebLogic developer install image by creating a sample WLS 12.2.1.3 domain and cluster to run in Kubernetes. The WebLogic domain consists of an Admininstrator Server and several Managed Servers running in a WebLogic cluster. All WebLogic servers share the same domain home which has been mapped to an external volume.
+
+## Prerequisites
+1. You need to have a Kubernetes cluster up and running with kubectl installed.
+2. You have built oracle/weblogic:12.2.1.3-developer image locally based on Dockerfile and scripts here: https://github.com/oracle/docker-images/tree/master/OracleWebLogic/dockerfiles/12.2.1.3/
+3. Username/password for the WebLogic domain are stored in k8s/secrets.yml and they are encoded by base64. The default values are weblogic/weblogic1.  
+If you want to customize it, first get the encoded data of your username/password via running `echo -n <username> | base64` and `echo -n <password> | base64`. Next upate k8s/secrets.yml with the new encoded data.
+
+## How to Build and Run
+
+### 1. Build the WebLogic Image for This Sample Domain
+Pre-steps before build the image:
+1. You need to download get-pip.py from https://bootstrap.pypa.io/get-pip.py and save it to folder 'container-scripts'.
+2. If you run `docker build` behind a proxy, you need to set up http and https proxy in the Dockerfile.
+
+Then build the image:
+```
+$ docker build -t wls-k8s-domain .
+```
+Or you can build the image by running build.sh directly.
```

Comment:
So `build.sh` just contains this exact command so there is no point in having it. I would just remove `build.sh` and this line completely.